tmt/utils.py

```diff
@@ -153,8 +153,73 @@ def unrooted(self) -> 'Path':
 # A stand-in variable for generic use.
 T = TypeVar('T')
 
-# A FMF context type, representing name/values context.
-FmfContextType = Dict[str, List[str]]
+
+class FmfContext(Dict[str, List[str]]):
+    """
+    Represents a fmf context.
```

Comment:
Well, maybe it is super obvious, but I would like to see mentioned that this is a fancy Dict with some additional class methods only? It does nto change the dict functionality so one can easily convert to ordinary dict with `dict(FmfContext)`?

Reply:
IMO this is obvious since it inherits from `dict`.

Reply:
ok ok chief